lib/zendesk_apps_support/assets/src.js.erb

```diff
@@ -1,38 +1,32 @@
-(function() {
```

Comment:
that's not the same, this was wrapping each individual app, not the whole of installed.js